File: pi/pi.py
import math
import time
num = 4.0
pi = 0
plus = True
startTime = time.time()
den = 1
while den < 100000000:
  if plus:
    pi = pi + num/den
    plus = False
  else:
    pi = pi - num/den
    plus = True  
  den = den + 2


print("PI = " + str(pi))              # calculated value of pi
print("Math::PI = " + str(math.pi))  # pi from the math class
endTime = time.time()
print(endTime - startTime)

File: pi/pi.rb
num = 4.0
pi = 0
plus = true
startTime = Time.now.getutc
den = 1
while den < 100000000
  if plus
    pi = pi + num/den
    plus = false
  else
    pi = pi - num/den
    plus = true
  end
  den = den + 2
end

puts "PI = #{pi}"              # calculated value of pi
puts "Math::PI = #{Math::PI}"  # pi from the math class
endTime = Time.now.getutc
puts endTime - startTime